DeadlineQos TestCase [6425] (#6)
eProsima/Shapes-Demo-Docs: docs/liveliness.rst

entities within the DomainParticipant alive.

> The remaining kinds (MANUAL_BY_PARTICIPANT and MANUAL_BY_TOPIC) need periodically an assertion to consider the remote 

Reviewer: Why did you remove this explanation? Did you consider it difficult to understand?

Author: I thought that explanation was too technical and since the test did not use any of those kinds, I decided to remove it. But if you think it can be understood correctly, I can add it again.

Reviewer: I think removing it is fine, I was just asking :)

participants as alive.

> In this test, we are going to create a Publisher and Subscriber using AUTOMATIC liveliness and a lease duration value

Reviewer: ```suggestion
In this test, we are going to create a publisher and subscriber using AUTOMATIC liveliness and a lease duration value
```

   - Start eProsima Shapes-Demo. (We will refer to this instance as Instance1)
   - Click on Publish.
   - Select SQUARE option for Shape and RED for Color.
   - Select AUTOMATIC.
   - Set Lease Duration to 150. (The default write rate is 75 ms)

2 - Create a square subscriber:
   - Start eProsima Shapes-Demo. (We will refer to this instance as Instance2)
   - Click on Subscribe.
   - Select SQUARE option for Shape.
   - Select AUTOMATIC.
   - Set a value for the Lease Duration higher or equal to the one stated for the Publisher. 
     (If the value of subscriber lease duration is lower the entities don't match)

.. image:: test8_1.png
   :scale: 60 %
   :alt: Initial state
   :align: center

If you go to the *Output Tab* on Instance2, you can observe that the subscriber has recovered the liveliness once it
matches with the publisher.

> Now, you have to open the Task Manager and kill the process corresponding with the publisher (Instnace1). As you can

Reviewer: ```suggestion
Now, open the Task Manager and kill the process corresponding to the publisher (Instnace1). As you can
```

> see, the subscriber lost the liveliness as the publisher doesn't terminate cleanly.

Reviewer: ```suggestion
see, the subscriber reported that liveliness was lost, as the publisher did not terminate cleanly.
```
